dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.javax.jaxrs-2.1.feature

```diff
@@ -6,8 +6,7 @@ singleton=true
 IBM-App-ForceRestart: uninstall, \
  install
 Subsystem-Name: Java RESTful Services API 2.1
--features=\
-  io.openliberty.servlet.api-4.0, \
+-features=io.openliberty.servlet.api-4.0, \
```

Comment:
It would be good to avoid touching the feature file if there is a single feature listed.